### DISCUSSION TITLE
v0.3.1.0

### DISCUSSION THREAD

**Discussion by isletennos** [isletennos/MMVC_Client#28] [Announcements]
・DirectML対応
・ONNX対応

<hr /><em>This discussion was created from the release <a href='https://github.com/isletennos/MMVC_Client/releases/tag/v0.3.1.0'>v0.3.1.0</a>.</em>

**Comment by marrie** [comment]
I notice that this will not work well on the radian GPU. I know this as my app cuts out below a second which leads me to believed that I'm running in CPU mode and the voice takes longer and longer to buffer even after I set it to a second. Is there a way to have this run on a radian GPU. I have a rizon processer running at 3.4ghz and a graphics radio card with 24 gigs of VRam. Thanks. Over all this is a very neat app.
